paymentsheet/src/main/java/com/stripe/android/paymentsheet/forms/CardSpec.kt

```diff
@@ -1,8 +1,47 @@
 package com.stripe.android.paymentsheet.forms
 
+import com.stripe.android.paymentsheet.R
+import com.stripe.android.paymentsheet.elements.CardBillingSpec
+import com.stripe.android.paymentsheet.elements.CardDetailsSpec
+import com.stripe.android.paymentsheet.elements.IdentifierSpec
 import com.stripe.android.paymentsheet.elements.LayoutSpec
 import com.stripe.android.paymentsheet.elements.SaveForFutureUseSpec
+import com.stripe.android.paymentsheet.elements.SectionSpec
+import com.stripe.android.paymentsheet.elements.billingParams
 
-internal val card = LayoutSpec.create(
+internal val CardRequirement = PaymentMethodRequirements(
+    piRequirements = emptySet(),
+    siRequirements = emptySet(),
+    confirmPMFromCustomer = true
+)
+
+internal val cardParams: MutableMap<String, Any?> = mutableMapOf(
+    "number" to null,
+    "exp_month" to null,
+    "exp_year" to null,
+    "cvc" to null,
+//    "attribution" to listOf("PaymentSheet.Form")
```

Comment:
Do we still need attribution here?

Reply:
No, we only care about the top-level `product_usage`. Any event from here would have the `PaymentSheet` tag, so this would be ignored anyway.

Reply:
No, we only care about the highest level `product_usage`. Any event from here would have the `PaymentSheet` tag, so this would be ignored anyway.